projects/addon-commerce/pipes/amount/amount.pipe.ts

```diff
@@ -35,7 +35,8 @@ export class TuiAmountPipe implements PipeTransform {
                         : format.precision,
                 });
                 const space =
-                    currencySymbol?.length > 1 || currencyAlign === 'right'
+                    currencySymbol &&
+                    (currencySymbol?.length > 1 || currencyAlign === 'right')
```

Comment:
Why this change?

Reply:
it was bug, when we added space without currencySymbol but with currencyAlign = 'right'